mypy.ini

```
@@ -21,3 +21,31 @@ warn_return_any = True
 [mypy-app.views.contexts.submit_context]
 disallow_untyped_defs = True
 warn_return_any = True
+
+[mypy-app.authentication.authenticator]
+disallow_untyped_defs = True
+warn_return_any = True
+
+[mypy-app.authentication.jti_claim_storage]
+disallow_untyped_defs = True
+warn_return_any = True
+
+[mypy-app.authentication.no_questionnaire_state_exception]
+disallow_untyped_defs = True
+warn_return_any = True
+
+[mypy-app.authentication.no_token_exception]
+disallow_untyped_defs = True
+warn_return_any = True
+
+[mypy-app.authentication.roles]
+disallow_untyped_defs = True
+warn_return_any = True
+
+[mypy-app.authentication.user]
+disallow_untyped_defs = True
+warn_return_any = True
+
+[mypy-app.authentication.user_id_generator]
+disallow_untyped_defs = True
+warn_return_any = True
```

When doing the entire module, this should just be:
```suggestion
[mypy-app.authentication.*]
disallow_untyped_defs = True
warn_return_any = True

```

Any file that can't be done, can be ignored at a file level.